src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/PositionExtensions.cs

```diff
@@ -52,6 +52,22 @@ public static bool TryGetAbsoluteIndex(this OmniSharpPosition position, SourceTe
             return TryGetAbsoluteIndex(position.Character, position.Line, sourceText, logger, out absoluteIndex);
         }
 
+        public static int CompareTo(this Position position, Position other)
+        {
+            if (position is null)
```

Comment:
Can we stop putting these in, now that we're mostly null annotated?

Reply:
I don't know that we ARE mostly null annotated (although I certainly aim to be). I'm doing it to be defensive for now, but how do people feel about this going forward? Do we wanna stay adding them and come back to remove them when we're more nullable, or draw a line and say it's the callers responsibility?

Reply:
I'm definitely on the "leave them out" side of things, and particularly for new methods, where all callers are (hopefully!) annotated I think it doesn't make sense to include them. But I just checked and we are only about 54% annotated so we can wait a little longer if people prefer.